x11-misc/mgm/mgm-1.2-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Moaning Goat Meter: load and status meter written in Perl"
+HOMEPAGE="http://www.linuxmafia.com/mgm"
+SRC_URI="http://downloads.xiph.org/releases/mgm/${P}.tgz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND=">=dev-lang/perl-5.6.1
+	>=dev-perl/Tk-800.024"
+
+HTML_DOCS=( doc/ )
```

Comment:
did you try it? It has the same semantics meaning as `HTML_DOCS=( doc/* )`, but the latter cant be used, as it would do a failglob in global scope